### ISSUE TITLE
Can the same tb-gatway enable opc-ua and MQTT protocols to connect to external services?

### ISSUE THREAD

**Issue by BIT-masterpy** [thingsboard/thingsboard-gateway#57]
Hi, I want to connect two external services to support the MQTT and opc-ua protocol respectively. Can I enable both of these functions through configuration in a tb-gateway?
![2018-04-12_144807](https://user-images.githubusercontent.com/30205233/38660593-5d67e8ba-3e60-11e8-9fc9-39604a876990.png)

**Comment by mp-loki** [comment]
Yep, that's possible. Which version of TB Gateway are you using? The config file is different between 1.2.2 and 1.4

**Comment by mp-loki** [comment]
Please use 1.2.2 instead. 1.2.1 is more than a year old an multiple issues have been fixed since.
https://github.com/thingsboard/thingsboard-gateway/releases

**Comment by BIT-masterpy** [comment]
Thank you very much. In version 1.4, how to configure MQTT and OPCUA to start simultaneously?
![2018-04-13_135244](https://user-images.githubusercontent.com/30205233/38718949-1efe9cf0-3f22-11e8-950d-1c7bdee9e286.png)

**Comment by mp-loki** [comment]
`remoteConfiguration: false`
extensions:
- 
  id: "mqtt"
  type: "MQTT"
  extensionConfiguration: mqtt-config.json
-
  id: "opc-ua"
  **type: "OPC UA"**
  extensionConfiguration: opc-config.json

**Comment by BIT-masterpy** [comment]
thank you @mp-loki
